### PR TITLE
Kevinf July docs

### PR DESCRIPTION
**Pull request by kfansler**
Mostly Python samples with some random cleanup. Each commit is labeled.

### CODE REVIEW

File: source/index.html.md.erb [hunk 1]
```diff
@@ -1891,15 +1908,15 @@ Attachment attachment = smartsheet.sheetResources().attachmentResources().versio
...
+response = ss_client.Attachments.attach_new_version(
+    9283173393803140,       # sheet_id
+    0123456789123452,       # attachment_id
     ('ProgressReport.docx',
      open('/path/to/ProgressReport.docx', 'rb'),
      'application/msword')
 )
-# action.message == 'SUCCESS'
-# action.result is an Attachment object
+# response.message == 'SUCCESS'
```
Comment (seweil) [inline source/index.html.md.erb]: This is true for almost all update commands - not sure it's worth calling out

File: source/index.html.md.erb [hunk 2]
```diff
@@ -1891,15 +1908,15 @@ Attachment attachment = smartsheet.sheetResources().attachmentResources().versio
...
+# response.result is an Attachment object
```
Comment (seweil) [inline source/index.html.md.erb]: In other cases we show this like:
`updated_attachment = response.result`

**Comment by kfansler** [comment]
I was wondering about those. I’m not sure where they came from. That style is repeated in a few places, and for Python only.

Kevin Fansler | Smartsheet
Technical Writer - API & SDKs
Kevin.Fansler@Smartsheet.com

File: source/index.html.md.erb [hunk 3]
```diff
@@ -1958,15 +1975,15 @@ Attachment attachment = smartsheet.sheetResources().attachmentResources().versio
 ```
 
 ```python
-action = smartsheet.Attachments.attach_new_version(
-    sheetId,
-    attachmentId,
+response = ss_client.Attachments.attach_new_version(
+    9283173393803140,       # sheet_id
+    0123456789123456,       # attachment_id
```
Comment (seweil) [inline source/index.html.md.erb:1980]: This syntax (below) is unfamiliar to me. Please test

File: source/index.html.md.erb [hunk 4]
```diff
@@ -1958,15 +1975,15 @@ Attachment attachment = smartsheet.sheetResources().attachmentResources().versio
...
     ('ProgressReport.docx',
      open('/path/to/ProgressReport.docx', 'rb'),
      'application/msword')
 )
-# action.message == 'SUCCESS'
-# action.result is an Attachment object
+# response.message == 'SUCCESS'
```
Comment (seweil) [inline source/index.html.md.erb]: See earlier comment about response.message and result

File: source/index.html.md.erb [hunk 5]
```diff
@@ -3222,13 +3249,17 @@ PagedResult<Column> columns = smartsheet.sheetResources().columnResources().list
...
+response = ss_client.Sheets.get_columns(
+    9283173393803140,       # sheet_id
+    include_all=True)
+columns = response.data
 
-# Paginate the list of columns (100 columns per page)
-action = smartsheet.Sheets.get_columns(sheetId)
-pages = action.total_pages
-columns = action.data
+# Sample 2: Paginate the list (100 columns per page)
```
Comment (seweil) [inline source/index.html.md.erb]: This example doesn't specify the paging params. Should either be explicit, or simply omit. (Column list is unlikely to be too large to get in one gulp)

File: source/index.html.md.erb [hunk 6]
```diff
@@ -3347,16 +3378,19 @@ Column updatedColumn = smartsheet.sheetResources().columnResources().updateColum
 
 ```python
 # Specify column properties
-altered_column = smartsheet.models.Column({
+altered_column = ss_client.models.Column({
```
Comment (seweil) [inline source/index.html.md.erb]: Elsewhere we call this `updated_column`

File: source/index.html.md.erb [hunk 7]
```diff
@@ -3622,14 +3656,14 @@ smartsheet.sheetResources().discussionResources().commentResources().addCommentW
...
     'text': 'You might want this.'
 })
 
 # Add comment (with attachment) to discussion
-action = smartsheet.Discussions.add_comment_to_discussion_with_attachment(
-    sheetId,
-    discussionId,
+response = ss_client.Discussions.add_comment_to_discussion_with_attachment(
+    2252168947361668,           # sheet_id
+    3962273862576004,           # discussion_id
```
Comment (seweil) [inline source/index.html.md.erb:3660]: Please test

File: source/index.html.md.erb [hunk 8]
```diff
@@ -3971,13 +4018,14 @@ PagedResult<Contact> contacts = smartsheet.contactResources().listContacts(
 ```
 
 ```python
-action = smartsheet.Contacts.list_contacts(include_all=True)
-contacts = action.data
+# Sample 1: List all
+response = ss_client.Contacts.list_contacts(include_all=True)
+contacts = response.data
 
-# Paginate the list of contacts (100 contacts per page)
```
Comment (seweil) [inline source/index.html.md.erb:3977]: Please add explicit pagination params

File: source/index.html.md.erb [hunk 9]
```diff
@@ -3971,13 +4018,14 @@ PagedResult<Contact> contacts = smartsheet.contactResources().listContacts(
...
-action = smartsheet.Contacts.list_contacts()
-pages = action.total_pages
-contacts = action.data
+# Sample 2: Paginate the list (100 contacts per page)
```
Comment (seweil) [inline source/index.html.md.erb:4019]: Please add explicit pagination params

File: source/index.html.md.erb [hunk 10]
```diff
@@ -4886,13 +4951,14 @@ PagedResult<Favorite> results = smartsheet.favoriteResources().listFavorites(
...
-favs = action.data
+# Sample 1: List all
+response = ss_client.Favorites.list_favorites(include_all=True)
+faves = response.data
 
-# Paginate the list
-action = smartsheet.Favorites.list_favorites(page_size=10, page=1)
-pages = action.total_pages
-favs = action.data
+# Sample 2: Paginate the list
```
Comment (seweil) [inline source/index.html.md.erb]: Correct, but unnecessary. Unlikely to have a huge # of favorites

File: source/index.html.md.erb [hunk 11]
```diff
@@ -4931,51 +4997,6 @@ Gets a list of all of the user's Favorite items.
 
 ## Remove Favorite
 
-### Remove Favorite Sight
-
-> Example Request: Remove Favorite Sight
-
-```shell
-curl https://api.smartsheet.com/2.0/favorites/sights/{sightId} \
```
Comment (seweil) [inline source/index.html.md.erb]: Why is this section removed?

**Comment by kfansler** [comment]
Just re-alphabetized. It was in the wrong place in the TOC.

Kevin Fansler | Smartsheet
Technical Writer - API & SDKs
Kevin.Fansler@Smartsheet.com

File: source/index.html.md.erb [hunk 12]
```diff
@@ -6053,8 +6130,9 @@ Folder folder = smartsheet.folderResources().getFolder(
 ```
 
 ```python
-folder = smartsheet.Folders.get_folder(folderId)
-# folder is an instance of smartsheet.models.Folder
+folder = ss_client.Folders.get_folder(
+    7960873114331012)       # folder_id
+# folder is an instance of ss_client.models.Folder
```
Comment (seweil) [inline source/index.html.md.erb]: Please delete this comment

File: source/index.html.md.erb [hunk 13]
```diff
@@ -14194,13 +14395,14 @@ PagedResult<User> users = smartsheet.userResources().listUsers( new HashSet(Arra
...
+# Sample 1; List all
+response = ss_client.Users.list_users(include_all=True)
+users = response.data
 
-# Paginate the list
-action = smartsheet.Users.list_users()
-pages = action.total_pages
-users = action.data
+# Sample 2: Paginate the list
+response = ss_client.Users.list_users()
```
Comment (seweil) [inline source/index.html.md.erb]: Add params

File: source/index.html.md.erb [hunk 14]
```diff
@@ -8752,32 +8862,21 @@ List<Row> updatedRows = smartsheet.sheetResources().rowResources().updateRows(
...
-cell_a.value = 'new value'
-cell_b = row_a.get_column(1888812600190852)
-cell_b.value = 'A'
-
-# use row set_column(id, Cell) convenience method
-# to replace the cell in-place
-row_a.set_column(cell_a.column_id, cell_a)
-row_a.set_column(cell_b.column_id, cell_b)
+# Build new cell value
+new_cell_spec = ss_client.models.Cell()
```
Comment (seweil) [inline source/index.html.md.erb]: This is a legitimate `cell` object. So I'd call it `new_cell`. I prefer "_spec" for the helper objects that provide info, but are a different type. Same for `new_row_spec` below. (P2)

**Pull request merged**